src/Faker/Provider/no_NO/Address.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Faker\Provider\no_NO;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $buildingNumber = array('%###', '%##', '%#', '%#?', '%', '%?');
+
+    protected static $streetPrefix = array(
+        "Øvre", "Nedre", "Søndre", "Gamle", "Østre", "Vestre"
+    );
+
+    protected static $streetSuffix = array(
+        "alléen", "bakken", "berget", "bråten", "eggen", "engen", "ekra", "faret", "flata", "gata", "gjerdet", "grenda", "gropa", "hagen", "haugen", "havna", "holtet", "høgda", "jordet", "kollen", "kroken", "lia", "lunden", "lyngen", "løkka", "marka", "moen", "myra", "plassen", "ringen", "roa", "røa", "skogen", "skrenten", "spranget", "stien", "stranda", "stubben", "stykket", "svingen", "tjernet", "toppen", "tunet", "vollen", "vika", "åsen"
+    );
+
+    protected static $streetSuffixWord = array(
+        "sgate", "svei", "s Gate", "s Vei", "gata", "veien"
+    );
+
+    protected static $postcode = array("####", "####", "####", "0###");
+
+    /**
+    * @var array Norwegian city names
+    * @link https://no.wikipedia.org/wiki/Liste_over_norske_byer
+    */
+    protected static $cityNames = array(
+        "Alta", "Arendal", "Askim", "Bergen", "Bodø", "Brekstad", "Brevik", "Brumunddal", "Bryne", "Brønnøysund", "Drammen", "Drøbak", "Egersund", "Elverum", "Fagernes", "Farsund", "Fauske", "Finnsnes", "Flekkefjord", "Florø", "Fosnavåg", "Fredrikstad", "Førde", "Gjøvik", "Grimstad", "Halden", "Hamar", "Hammerfest", "Harstad", "Haugesund", "Hokksund", "Holmestrand", "Honningsvåg", "Horten", "Hønefoss", "Jessheim", "Jørpeland", "Kirkenes", "Kolvereid", "Kongsberg", "Kongsvinger", "Kopervik", "Kragerø", "Kristiansand", "Kristiansund", "Langesund", "Larvik", "Leknes", "Levanger", "Lillehammer", "Lillesand", "Lillestrøm", "Lyngdal", "Mandal", "Mo i Rana",  "Moelv", "Molde", "Mosjøen", "Moss", "Mysen", "Måløy", "Namsos", "Narvik", "Notodden", "Odda", "Orkanger", "Oslo", "Otta", "Porsgrunn", "Risør", "Rjukan", "Røros", "Sandefjord", "Sandnes", "Sandnessjøen", "Sandvika", "Sarpsborg", "Sauda", "Ski", "Skien", "Skudeneshavn", "Sortland", "Stathelle", "Stavanger", "Stavern", "Steinkjer", "Stjørdalshalsen", "Stokmarknes", "Stord", "Svelvik", "Svolvær", "Tromsø", "Trondheim", "Tvedestrand", "Tønsberg", "Ulsteinvik", "Vadsø", "Vardø", "Verdalsøra", "Vinstra", "Åkrehamn", "Ålesund", "Åndalsnes", "Åsgårdstrand"
```

Comment:
Could you split that to several lines? Same for other very long lines. Eases review.